File: app/src/main/java/com/coderpig/drysisters/utils/ToastUtils.java
package com.coderpig.drysisters.utils;

import android.view.Gravity;
import android.widget.Toast;

import com.coderpig.drysisters.DrySisterApp;

/**
 * 描述：Toast 工具类
 *
 * @author CoderPig on 2018/02/14 11:06.
 */

public class ToastUtils {
    public static void shortToast(String msg) {
        Toast toast = Toast.makeText(DrySisterApp.getContext(),msg, Toast.LENGTH_SHORT);
        toast.setGravity(Gravity.BOTTOM, 0, 40);
        toast.show();
    }

    public static void longToast(String msg) {
        Toast toast = Toast.makeText(DrySisterApp.getContext(),msg, Toast.LENGTH_LONG);
        toast.setGravity(Gravity.BOTTOM, 0, 40);
        toast.show();
    }
}

File: cp-base/src/main/java/cn/coderpig/cp_base/utils/ToastUtils.kt
package cn.coderpig.cp_base.utils

import android.widget.Toast
import cn.coderpig.cp_base.CpKit

/**
 * Author: zpj
 * Date: 2023-09-25 16:24
 * Desc: Toast相关工具类
 */

fun shortToast(msg: String) = Toast.makeText(CpKit.appContext, msg, Toast.LENGTH_SHORT).show()

fun longToast(msg: String) = Toast.makeText(CpKit.appContext, msg, Toast.LENGTH_LONG).show()